plugins/modules/copr.py

```diff
@@ -103,6 +103,29 @@
 from ansible.module_utils.basic import AnsibleModule  # pylint: disable=import-error
 from ansible.module_utils.urls import open_url  # pylint: disable=import-error
 
+HAS_RESPAWN_UTIL = False
+if not HAS_DNF_PACKAGES:
```

Comment:
Is there a reason why you only import this conditionally?

Reply:
Yeah, that's probably not necessary.

Reply:
https://github.com/ansible-collections/community.general/pull/6522/commits/934389f9fbafceae5d3f271413b2618a2ab2c263